src/Diagnostics.ModelsAndUtils/Models/AppInsights/AppInsightsTag.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Diagnostics.ModelsAndUtils.Models
+{
+    public class AppInsightsTag
```

Comment:
Question here - Do we use this model in the detector code somewhere? If not, you can potentially just move this class to the project where its used (Data Provider project).

Reply:
Nope here only....will make the chang